src/database/sqlite/sqlite_functions.c

```diff
@@ -410,8 +410,20 @@ int sqlite_library_init(void)
 
 SPINLOCK sqlite_spinlock = SPINLOCK_INITIALIZER;
 
+int sqlite_release_memory(int bytes)
+{
+    return sqlite3_release_memory(bytes);
+}
+
 void sqlite_library_shutdown(void)
 {
+#ifdef NETDATA_INTERNAL_CHECKS
+    int bytes;
+    do {
+        bytes = sqlite_release_memory(1024 * 1024);
```

Comment:
In description you wrote `If compiled with NETDATA_INTERNAL_CHECKS, agent with release memory during shutdown in 1MB chunks`, but here we have Kilobytes. 

Only to be sure we are not missing a constant here, is description right?